metadata_tools/label_support.py

```
@@ -10,16 +10,19 @@
 import metadata_tools.defs as defs
 
 #===============================================================================
-def create(filepath, system=None,
+def create(filepath, host_template_path, 
+                     system=None,
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)